gitpod-start-app.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
```

Comment:
I put this in a script as I'm pretty sure it was the only way to get `eval $(gp env -e)` to be recognized by `bin/startup`.